examples/animate-stroke.html

```diff
@@ -51,6 +52,7 @@
               fresh.total += d;
             });
 
+            var rect = fresh.getBoundingClientRect();
```

Comment:
New Two.js now takes paths' endings into account when calculating their bounds, whereas old Two.js didn't seem to. This would cause a feedback loop if the bounding box was fetched every time the path's ending changed.